shopfloor_packing/services/packaging.py

```diff
@@ -0,0 +1,75 @@
+# Copyright 2020-2021 Camptocamp SA (http://www.camptocamp.com)
+# Copyright 2020-2021 Jacques-Etienne Baudoux (BCIM) <je@bcim.be>
+# Copyright 2024 ACSONE SA/NV (https://www.acsone.eu)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+
+from odoo.addons.component.core import Component
+from odoo.addons.stock.models.stock_picking import Picking
+
+
+class PackagingAction(Component):
+
+    _inherit = "shopfloor.packaging.action"
```

Comment:
Just to not confuse with product packaging
```suggestion
class PackAction(Component):

    _inherit = "shopfloor.pack.action"
```